internal/tablemap.go

```diff
@@ -127,3 +127,43 @@ func GetSpannerCols(conv *Conv, srcTable string, srcCols []string) ([]string, er
 	}
 	return spCols, nil
 }
+
```

Comment:
I agree this is the right place/file to put this function.
However, the name of the file is a bit misleading now, since it does more than table maps.
Maybe rename to "mapping.go"?

Reply:
renamed file to mapping.go and mapping_test.go